pkg/management/component.go

```diff
@@ -79,6 +80,7 @@ type KeycloakClient interface {
 	LinkShadowUser(accessToken string, realmName string, userID string, provider string, fedID kc.FederatedIdentityRepresentation) error
 	ClearUserLoginFailures(accessToken string, realmName, userID string) error
 	GetAttackDetectionStatus(accessToken string, realmName, userID string) (map[string]interface{}, error)
+	GetIdps(accessToken string, realmName string) ([]kc.IdentityProviderRepresentation, error)
```

Comment:
Why don't we name this function GetIdentityProviders as we usually have full names in our functions?

Reply:
It is the naming in the keycloak-client